integration-tests/tests/ubuntuFan_test.go

```diff
@@ -1,5 +1,5 @@
 // -*- Mode: Go; indent-tabs-mode: t -*-
-// +build !excludeintegration
+// +build !excludeintegration,!excludeflaky
```

Comment:
Currently some tests in the fanTestSuite give inconsistent results when executed on canonistack instances, sometimes they fail as in [1] (you'll need to have the VPN up to access the server, search for `FAIL:` to see the actual error). This error doesn't appear in all the executions, only eventually on canonistack instances and never in other platforms (we get solid good results on kvm and the boards).

We are not going to use these instances for CI purposes, but someone executing the suite in a slow system may hit this errors (all the suites are executed by default), IMO it is good to have it tagged. For the rest of 'normal' systems the tests provide the intended value, covering parts of the product functionality.

About the failure itself, the problem happens when, after reconfiguring docker, we restart the daemon [2]. We try to wait the service to be reported as active by systemctl and the socket file to be present in /run, but sometimes one of these things don't happen (no matter how long we wait). We had similar problem before [3], that time modifying the daemon configuration solved the issue.

I've changed the tag to `lowperformance`, let me know if you thing we should drop the suite, as mentioned it is unreliable only in a few, very specific systems and usually works fine.

[1] http://10.55.60.58:8080/job/snappy-cloud-test/4/consoleFull
[2] https://github.com/fgimenez/snappy/blob/master/integration-tests/tests/ubuntuFan_test.go#L182
[3] https://bugs.launchpad.net/snappy/+bug/1482685